### PR TITLE
docs(resource-detector-aws): update readme for aws resource detector

### PR DESCRIPTION
**Pull request by maryliag**
## Which problem is this PR solving?

- Part Of #2055
- Part Of #2025

## Short description of the changes

- Update README of AWS Resource Detector to clarify the source of the data and add Semantic Conventions.

### CODE REVIEW

File: detectors/node/opentelemetry-resource-detector-aws/README.md [hunk 1]
```diff
@@ -31,15 +31,86 @@ const resource = await detectResources({
...
-- `awsEcsDetector`: Populates `container` for containers running on [Amazon ECS](https://aws.amazon.com/ecs/)
-- `awsEksDetector`: Populates `container` and `k8s.cluster_name` for containers running on [Amazon EKS](https://aws.amazon.com/eks/)
-  - `k8s.cluster_name` is not always available depending on the configuration of CloudWatch monitoring for the EKS cluster
-- `awsLambdaDetector`: Populates `faas` and `cloud` for functions running on [AWS Lambda](https://aws.amazon.com/lambda/)
-  - `faas.id` is currently not populated as it is not provided by the runtime at startup
+## Available detectors & Semantic Conventions
+
+This package uses `@opentelemetry/semantic-conventions` version `1.22+`, which implements Semantic Convention [Version 1.7.0](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.7.0/semantic_conventions/README.md)
+
+### Aws Beanstalk Detector
```
Comment (JamieDanielson) [inline detectors/node/opentelemetry-resource-detector-aws/README.md:38]:
```suggestion
### AWS Beanstalk Detector
```

File: detectors/node/opentelemetry-resource-detector-aws/README.md [hunk 2]
```diff
@@ -31,15 +31,86 @@ const resource = await detectResources({
...
+| Resource Attribute  | Description                                                              | Notes                                  |
+|---------------------|--------------------------------------------------------------------------| -------------------------------------- |
+| cloud.platform      | The cloud platform. In this context, it's always "aws_elastic_beanstalk" | Key: `SEMRESATTRS_CLOUD_PLATFORM`      |
+| cloud.provider      | The cloud provider. In this context, it's always "aws"                   | Key: `SEMRESATTRS_CLOUD_PROVIDER`      |
+| service.instance.id | Value of `deployment_id` from config file `environment.conf`             | Key: `SEMRESATTRS_SERVICE_INSTANCE_ID` |
+| service.name        | The service name. In this context, it's always "aws_elastic_beanstalk"   | Key: `SEMRESATTRS_SERVICE_NAME`        |
+| service.namespace   | Value of `environment_name` from config file `environment.conf`          | Key: `SEMRESATTRS_SERVICE_NAMESPACE`   |
+| service.version     | Value of `version_label` from config file `environment.conf`             | Key: `SEMRESATTRS_SERVICE_VERSION`     |
+
+### AWS Ec2 Detector
```
Comment (JamieDanielson) [inline detectors/node/opentelemetry-resource-detector-aws/README.md:51]:
```suggestion
### AWS EC2 Detector
```

File: detectors/node/opentelemetry-resource-detector-aws/README.md [hunk 3]
```diff
@@ -31,15 +31,86 @@ const resource = await detectResources({
...
+| cloud.availability_zone | Value of `availabilityZone` from `/latest/dynamic/instance-identity/document` request | Key: `SEMRESATTRS_CLOUD_AVAILABILITY_ZONE` |
+| cloud.platform          | The cloud platform. In this context, it's always "aws_ec2"                            | Key: `SEMRESATTRS_CLOUD_PLATFORM`          |
+| cloud.provider          | The cloud provider. In this context, it's always "aws"                                | Key: `SEMRESATTRS_CLOUD_PROVIDER`          |
+| cloud.region            | Value of `region` from `/latest/dynamic/instance-identity/document` request           | Key: `SEMRESATTRS_CLOUD_REGION`            |
+| host.id                 | Value of `instanceId` from `/latest/dynamic/instance-identity/document` request       | Key: `SEMRESATTRS_HOST_ID`                 |
+| host.name               | Value of `hostname` from `/latest/dynamic/instance-identity/document` request         | Key: `SEMRESATTRS_HOST_NAME`               |
+| host.type               | Value of `instanceType` from `/latest/dynamic/instance-identity/document` request     | Key: `SEMRESATTRS_HOST_TYPE`               |
+
+### AWS Ecs Detector
```
Comment (JamieDanielson) [inline detectors/node/opentelemetry-resource-detector-aws/README.md:66]:
```suggestion
### AWS ECS Detector
```

File: detectors/node/opentelemetry-resource-detector-aws/README.md [hunk 4]
```diff
@@ -31,15 +31,86 @@ const resource = await detectResources({
...
+| cloud.account.id        | Parsed value from the `TaskARN` | Key: `SEMRESATTRS_CLOUD_ACCOUNT_ID`        |
+| cloud.availability_zone | Value of `AvailabilityZone` from `ECS_CONTAINER_METADATA_URI_V4/task` request. This value is not available in all Fargate runtimes | Key: `SEMRESATTRS_CLOUD_AVAILABILITY_ZONE` |
+| cloud.platform          | The cloud platform. In this context, it's always "aws_ecs"                             | Key: `SEMRESATTRS_CLOUD_PLATFORM`          |
+| cloud.provider          | The cloud provider. In this context, it's always "aws"                                 | Key: `SEMRESATTRS_CLOUD_PROVIDER`          |
+| cloud.region            | Parsed value from the `TaskARN`                                                        | Key: `SEMRESATTRS_CLOUD_REGION`            |
+| container.id            | Value of from file `/proc/self/cgroup`                                                 | Key: `SEMRESATTRS_CONTAINER_ID`            |
+| container.name          | The hostname of the operating system                                                   | Key: `SEMRESATTRS_CONTAINER_NAME`          |
+
+### AWS Eks Detector
```
Comment (JamieDanielson) [inline detectors/node/opentelemetry-resource-detector-aws/README.md:90]:
```suggestion
### AWS EKS Detector
```

**Review by JamieDanielson — COMMENTED** [review]
This is looking great! I left a few nits around naming capitalization, but at a glance it seems really helpful. I don't know a ton about the attributes to confirm the descriptions.

**Comment by maryliag** [comment]
updates made and also remove the notes column

**Review by trentm — APPROVED** [review]

**Pull request merged**